src/TouristInfoCards.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const TouristInfoCards = (props) => {
+  return( 
+    <div className="tourist-cards">
+
+        {props.cardAttributes.map((attribute, index) => (
+            <div className="card" key ={index}>
+    <img src={attribute.img} className="card-img-top" alt= "{attribute.city} city" />
```

Comment:
We'd normally indent things so that children (like this `img` tag) are further right than their parents (the containing `div` tag) - doesn't change the functionality, but makes it a little easier to read the code :)